platform/ios/src/MGLMapView.mm

```diff
@@ -636,6 +636,10 @@ - (void)createGLView
     _glView.layer.opaque = _opaque;
     _glView.delegate = self;
 
+    // `CAEAGLLayer.presentsWithTransaction` was introduced in iOS 9.
```

Comment:
Since this project requires iOS 9, it feels unnecessary to include this comment (as it’s implied globally).

Reply:
Left for @jmkiley

Reply:
Would it be more useful to keep track of backwards compatibility at the PR-level — as a project, perhaps? This comment hints at the larger change here, but doesn’t help a reader outside of the context of this PR.

Reply:
Agreed. Will discuss.